### PR TITLE
[DOCS] Changes feature importance links to point to the new page

### PR DESCRIPTION
**Pull request by szabosteve**
## Overview

This PR updates the links pointing to feature importance so that they open the new, dedicated page of the ML book.

## Dependency note

This PR depends on changes of https://github.com/elastic/stack-docs/pull/1004. The docs CI will not pass until that PR is merged.

### CODE REVIEW

File: docs/reference/ml/df-analytics/apis/put-dfanalytics.asciidoc [hunk 1]
```diff
@@ -158,9 +158,8 @@ categories, the API reports all category probabilities. Defaults to 2.
 `num_top_feature_importance_values`::::
 (Optional, integer)
 Advanced configuration option. Specifies the maximum number of
-{ml-docs}/dfa-classification.html#dfa-classification-feature-importance[feature
-importance] values per document to return. By default, it is zero and no feature importance
-calculation occurs.
+{ml-docs}/{ml-docs}/ml-feature-importance.html[{feat-imp}] values per
```
Comment (szabosteve) [inline docs/reference/ml/df-analytics/apis/put-dfanalytics.asciidoc:161]:
```suggestion
{ml-docs}/ml-feature-importance.html[{feat-imp}] values per 
```

**Review by lcawl — APPROVED** [review]
LGTM

**Comment by szabosteve** [comment]
@elasticmachine test this please

**Comment by szabosteve** [comment]
**Backport commits:**
7.x: a5cf471
7.7: 5c928ef

**Pull request merged**